src/components/Stat/Stat.stories.tsx

```diff
@@ -0,0 +1,43 @@
+import { Meta, StoryObj } from "@storybook/react"
+
+import StatComponent from "."
+
+const meta = {
+  title: "Molecules / Display Content / Stat",
+  component: StatComponent,
+} satisfies Meta<typeof StatComponent>
+
+export default meta
+
+export const Stat: StoryObj<typeof meta> = {
+  args: {
+    tooltipProps: {
+      content: "Tooltip content",
+    },
+    isError: false,
+    label: "Label",
+    value: "000",
+  },
+  argTypes: {
+    isError: {
+      table: {
+        disable: true,
+      },
+    },
+  },
+}
```

Comment:
That table option simply disables the ability to toggle the `isError` value in the storybook dashboard. It does not impact the render itself.

So I disabled it for both stories because it is not practical to toggle this prop in testing when the rendering for prod is based on the data coming in, or the data failing to come in.